src/app/kbv/controllers/load-question.test.js

```diff
@@ -0,0 +1,130 @@
+const BaseController = require("hmpo-form-wizard").Controller;
+
+const proxquire = require("proxyquire");
+const reqres = require("reqres");
+const WizardModel = require("hmpo-form-wizard/lib/wizard-model.js");
+const JourneyModel = require("hmpo-form-wizard/lib/journey-model");
+
+describe("Load Question controller", () => {
+  let LoadQuestionController;
+  let loadQuestionController;
+
+  let req;
+  let res;
+  let next;
+
+  let axiosStub;
+
+  beforeEach(() => {
+    axiosStub = {
+      get: sinon.stub(),
+    };
+
+    LoadQuestionController = proxquire("./load-question", {
+      axios: axiosStub,
+      "../../../lib/config": {
+        API: {
+          BASE_URL: "http://example.com",
+          PATHS: { QUESTION: "/q" },
+        },
+      },
+    });
+
+    req = reqres.req();
```

Comment:
From here we can go and call the `global.setupReqResNext()` to save lines of code 🤓

Reply:
:scissors: